app/code/community/Businessfactory/Roihuntereasy/Model/Cron.php

```diff
@@ -24,16 +24,20 @@ public function scheduleFeed($schedule) {
      */
     public function createFeed($fileFormat)
     {
-        Mage::log($fileFormat . ": "  . __METHOD__ . " cron", null, 'cron.log');
+        if (!isset($fileFormat) || trim($fileFormat)==='' ) {
```

Comment:
In my point of view this is not good practice for getting default value, because method behavior is not clear at first sight. In this case you should resolve default value when you get format from $jobConfig.